src/python/tests/fasta/test_fasta_prep_data.py

```diff
@@ -0,0 +1,149 @@
+# See the NOTICE file distributed with this work for additional information
+# regarding copyright ownership.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Unit testing of :mod:`ensembl.io.genomio.fasta.process` module.
+
+"""
+
+import filecmp
+from pathlib import Path
+from typing import List
```

Comment:
```suggestion
from contextlib import nullcontext as does_not_raise
import filecmp
from pathlib import Path
from typing import ContextManager, List, Set
```
See suggestion below to understand this suggestion.